javascript/ecmascript-2015/promises/how-to-create-promises.md

```diff
@@ -53,31 +46,25 @@ let p = new Promise((resolve, reject) => {
   setTimeout(() => {
     resolve("enki");
   }, 2000);
-})
-.then((result) => {
- console.log(result)
+}).then(result => {
+  console.log(result);
 });
 
 // waits 2 seconds
 // logs "enki"
 ```
 
-We declared a new promise supplying a function with two arguments resolve and reject.
-
-Next we used `setTimeout` to asynchronously call the resolve function after two seconds passing the value `"enki"`.
-
-When resolve was called after two seconds we then logged this value to the console.
+We declared a new promise supplying a function with two arguments resolve and reject. Next we used `setTimeout` to asynchronously call the resolve function after two seconds passing the value `"enki"`. When resolve was called after two seconds we then logged this value to the console.
```

Comment:
put resolve and reject in code? `resolve` and `reject`

Reply:
done